Makefile

```diff
@@ -45,4 +45,4 @@ vis:
 	$(GIT_SUBTREE) --prefix=lib/vis https://github.com/almende/vis.git master
 
 typeahead:
```

Comment:
Since you touched this file, can you add typehead to the `update-subtrees` target as well? :)